buildbot/cmd/buildbot/main.go

```diff
@@ -62,16 +48,15 @@ func main() {
 		if user.Name == "vdemeest" {
 			vdemeest = user.ID
 		}
-		if _, ok := cops[user.Name]; ok {
-			copsID[user.Name] = user.ID
-		}
+		copsID[user.Name] = user.ID
 	}
 
-	currentCop = getBuildCop(time.Now())
+	r := NewRotationFromURL(rotationURL)
```

Comment:
If we copied this into `dailyPing`'s for loop, would we also get updates pulled in every day to the csv?  Any reason not to do that?

Reply:
nm!  I realised that calling r.GetBuildCop(time.Now()) makes a request for a new version of the CSV.  Nice!